apps/wallet-mobile/package.json

```diff
@@ -4,14 +4,20 @@
   "private": true,
   "scripts": {
     "android-bundle": "react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res",
-    "e2e:appium-test-android": "./node_modules/.bin/wdio tests/config/wdio.android.local.conf.ts --suite happyPathTx",
-    "e2e:appium-test-android-ledger": "./node_modules/.bin/wdio tests/config/wdio.android.local.conf.ts --suite ledger",
-    "e2e:build-android": "detox build -c android.emu.mainnet.release",
-    "e2e:build-android-debug": "detox build -c android.emu.mainnet.debug",
-    "e2e:build-ios": "detox build -c ios.release",
-    "e2e:test-android": "detox test -c android.emu.mainnet.release",
-    "e2e:test-android-debug": "detox test -c android.emu.mainnet.debug",
-    "e2e:test-ios": "detox test -c ios.release",
+    "e2e:build:android": "detox build -c android.emu.mainnet.release",
+    "e2e:test:android": "detox test -c android.emu.mainnet.release",
+    "e2e:build:android:debug": "detox build -c android.emu.mainnet.debug",
+    "e2e:test:android:debug": "detox test -c android.emu.mainnet.debug",
+    "e2e:build:android:dev:debug": "detox build --configuration android.emu.debug",
+    "e2e:test:android:dev:debug": "detox test --configuration android.emu.debug",
+    "e2e:build:android:nightly:release": "detox build --configuration android.emu.nightly.release",
+    "e2e:test:android:nightly:release": "detox test --configuration android.emu.nightly.release",
```

Comment:
This part is outdated. We can remove it.

Reply:
Done